lib/dealership.rb

```diff
@@ -0,0 +1,47 @@
+class Dealership
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def inventory_count
+    @inventory_count = @inventory.length
```

Comment:
You do not need to create this extra instance variable called `@inventory_count`. This method will return that for your with the `inventory.length`.